packages/x-data-grid/src/locales/ptPT.ts

```diff
@@ -55,7 +56,7 @@ const ptPTGrid: Partial<GridLocaleText> = {
   columnsManagementNoColumns: 'Sem colunas',
   columnsManagementShowHideAllText: 'Mostrar/Ocultar Todas',
   columnsManagementReset: 'Repor',
-  // columnsManagementDeleteIconLabel: 'Clear',
+  columnsManagementDeleteIconLabel: 'Limpar',
```

Comment:
@oliviertassinari During the pre-release phase, we cherry-pick every `l10n` change to the current stable branch. 😉

Reply:
This is a generous policy. Alright, done with https://github.com/mui/mui-x/commit/5b9f974d1af80d339f4d63fc96440aecace50c01.